about.html

```diff
@@ -7,66 +7,122 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
+  <link rel="stylesheet" href="about.html">
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
+<div class="space">
+  <img class="lamdaimg" src="img/lambda-black.png">
+    
+    <nav>
+      <div> <a href="index.html">Home</a></div>
+      <div> <a href="about.html">About</a></div>
+      <div><a href="#products"> Products</a></div>
+      <div> <a href="#"> Blog</a></div>
+      <div><a href="#"> Contact</a></div>
+    </nav>
+    
+    </div>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <Section class="bodyimghead">
+      <img src="./img/jumbo-about.png">
 
+    </Section>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+<body>
+    <div class="containerabout-page">
+     
+    <h2>About</h2>
 
-    Learn More
+    <p>Leverage agile frameworks to provide a robust synopsis for high 
+      level overviews. Iterative approaches to corporate strategy foster
+       collaborative thinking to further the overall value proposition.
+        Organically grow the holistic world view of disruptive innovation 
+        via workplace diversity and empowerment.</p>
+    </div>
 
+    
+<!--this is the section with four imgs and buttons-->
 
```

Comment:
Good use of semantic HTML here using sections.  Commenting above also helps as well!